gradle/android_common.gradle

```diff
@@ -72,7 +70,7 @@ android {
 
     kotlinOptions {
         jvmTarget = "1.8"
-        freeCompilerArgs += ['-XXLanguage:+InlineClasses']
+        freeCompilerArgs += ['-XXLanguage:+InlineClasses', '-Xjvm-default=all']
```

Comment:
Now from the consumer side when `KalugaState` is extended, if this argument is not passed the build will throw:
```
Inheritance from an interface with '@JvmDefault' members is only allowed with -Xjvm-default option
```

Reply:
@JvmDefault is enforced by the upgrade, so yeah, projects need to add it as well. I cannot revert that.